app/src/main/java/com/ivy/wallet/ui/home/HomeTab.kt

```diff
@@ -160,12 +166,28 @@ private fun BoxWithConstraintsScope.UI(
         mutableStateOf(null)
     }
     var expanded by remember { mutableStateOf(false) }
+    val offset by remember { mutableStateOf(Offset.Zero) }
 
     Column(
         modifier = Modifier
             .fillMaxSize()
             .statusBarsPadding()
+            .offset {
+                IntOffset(offset.x.roundToInt(), offset.y.roundToInt())
+            }
             .navigationBarsPadding()
+            .pointerInput(Unit) {
+                detectDragGestures(
```

Comment:
detectVerticalDragGesture() should be used and only swipe in downwards direction should be detected